Workbooks/WorkbooksMetadata.json

```diff
@@ -90,6 +90,20 @@
     "subtitle": "",
     "provider": "Palo Alto Networks"
   },
+  {
```

Comment:
This metadata addition should be removed, metadata changes are not necessary for solutions.